### PR TITLE
Twriter edits to k0s Configuration content.

### PR DESCRIPTION
**Pull request by KoryKessel-Mirantis**
Technical writer edits to k0s Configuration topics content.

### CODE REVIEW

File: docs/worker-node-config.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-
+**Note:** Setting the labels is only effective on the first registration of the node. Changing the labels thereafter has no effect.
 
 ## Kubelet args
 
-`k0s worker` command accepts a generic flag to pass in any set of argument for kubelet process.
+The `k0s worker` command accepts a generic flag to pass in any set of arguments for kubelet process.
 
-For example running `k0s worker --token-file=k0s.token --kubelet-extra-args="--node-ip=1.2.3.4 --address=0.0.0.0"` will "pass on" the given flags to kubelet as-is. As the flags are passed as-is make sure you are passing in properly formatted and valued flags as k0s will NOT validate those at all.
+For example, running 1k0s worker --token-file=k0s.token --kubelet-extra-args="--node-ip=1.2.3.4 --address=0.0.0.0"` passes in the given flags to kubelet as-is. As such, you must confirm that any flags you are passing in are properly formatted and valued as k0s will not validate those flags.
```
Comment (mviitane) [inline docs/worker-node-config.md:23]: 1k0s->k0s

**Review by mviitane — APPROVED** [review]
Looks good.

File: docs/configuration.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
-k0s Control plane can be configured via a YAML config file. By default `k0s controller` command reads a file called `k0s.yaml` but can be told to read any yaml file via `--config` option.
+You can configure the k0s Control plane using a YAML config file. By default, the `k0s controller` command reads a file named `k0s.yaml`, however you can but can set the command to read any yaml file with the `--config` option.
```
Comment (kke) [inline docs/configuration.md:1]: "you can but can"

File: docs/configuration.md [hunk 3]
```diff
@@ -79,88 +81,108 @@ spec:
...
 
-- `type`: Type of the data store, either `etcd` or `kine`.
-- `etcd.peerAddress`: Nodes address to be used for etcd cluster peering.
-- `kine.dataSource`: [kine](https://github.com/rancher/kine/) datasource URL.
-
-Using type `etcd` will make k0s to create and manage an elastic etcd cluster within the controller nodes.
+| Element   | Description           |
+|-----------|---------------------------|
+| `type`      | Type of the data store (valid values:`etcd` or `kine`).<br><br>**Note**: Type `etcd` will cause k0s to create and manage an elastic etcd cluster within the controller nodes.|   
+| `etcd.peerAddress`      | Nodes address used for etcd cluster peering.|
```
Comment (kke) [inline docs/configuration.md:105]:
```suggestion
| `etcd.peerAddress`      | Node's address used for etcd cluster peering.|   
```

File: docs/worker-node-config.md (hunk 1 above)
Comment (kke) [inline docs/worker-node-config.md:23]:
```suggestion
For example, running `k0s worker --token-file=k0s.token --kubelet-extra-args="--node-ip=1.2.3.4 --address=0.0.0.0"` passes in the given flags to kubelet as-is. As such, you must confirm that any flags you are passing in are properly formatted and valued as k0s will not validate those flags.
```

File: docs/configuration.md (hunk 3 above)
Comment (KoryKessel-Mirantis) [reply docs/configuration.md:105]: Actually, it is better as: "Node adress used for etcd clusger peering."

**Review by kke — APPROVED** [review]

**Pull request merged**